components/html-block/html-block.js

```diff
@@ -3,6 +3,43 @@ import { css, LitElement } from 'lit-element/lit-element.js';
 import { htmlBlockMathRenderer } from '../../helpers/mathjax.js';
 import { requestInstance } from '../../mixins/provider-mixin.js';
 
+let contextPromise;
+
+const getContext = () => {
+	if (contextPromise) return contextPromise;
+
+	contextPromise = new Promise(async resolve => { // eslint-disable-line no-async-promise-executor
+		const context = JSON.parse(document.documentElement.getAttribute('data-hb-context'));
+		if (window.ifrauclient) {
+			const ifrauClient = await getIfrauClient();
+			context.host = await ifrauClient.request('valenceHost');
```

Comment:
I wish I could not use ifrau at all, but @mentions requires links to profile cards within the LMS, which are unfortunately LMS routes and thus within FRAs definitely need to have the host pre-pended.

Right now @mentions isn't available in any FRAs, but that may change.

Reply:
One thing maybe worth considering is that if we slapped the `valenceHost` as a data attribute on the host, we could use the new marshaling to pull that into the iframe without needing to explicitly invoke the client.